.github/workflows/integration.yaml

```diff
@@ -58,6 +58,7 @@ jobs:
           provider: microk8s
           channel: 1.21/stable
 
+      - run: sg microk8s -c "microk8s enable metallb:'10.64.140.43-10.64.140.49,192.168.0.105-192.168.0.111'"
```

Comment:
If we get [this PR](https://github.com/charmed-kubernetes/actions-operator/pull/33) over the finish line, there will be a clean way to do this in the block above (doesn't have to happen to merge this, more just trying to spur discussion).  Would that PR make sense for you?

Reply:
Ah that's nice! That PR would help us here (and in other charms, istio-operators for instance)

Reply:
Yeah I remember it coming up months ago for istio (I think that's why I happened to comment on that open PR), but it never got merged.  The repo is kind of derelict now that Cory left and has no obvious maintainer - I'll message Adam